### PR TITLE
Sort apps by package name

### PR DESCRIPTION
**Pull request by TarasMazepa**
So users will get at least some predictability when they see list of icons :)

### CODE REVIEW

File: app/src/main/java/uk/co/nickbutcher/adaptiveiconplayground/MainActivity.kt [hunk 1]
```diff
@@ -298,7 +298,9 @@ class MainActivity : AppCompatActivity() {
             val pm = context.packageManager
             val adaptiveIcons = ArrayList<AdaptiveIconDrawable>()
             val launcherIntent = Intent().apply { addCategory(Intent.CATEGORY_LAUNCHER) }
-            pm.getInstalledApplications(0).forEach { appInfo ->
+            val apps = pm.getInstalledApplications(0)
+            apps.sortBy { it.packageName }
```
Comment (nickbutcher) [inline app/src/main/java/uk/co/nickbutcher/adaptiveiconplayground/MainActivity.kt]: Can you do this fluently i.e.

```
val apps = pm.getInstalledApplications(0)
    .sortBy { it.packageName }
    .forEach { appInfo -> 
    …
```

**Review by nickbutcher — CHANGES_REQUESTED** [review]
Thanks for the contribution, one small comment.

**Comment by TarasMazepa** [comment]
sortBy returns Unit. To do this fluently apply should be used :)

File: app/src/main/java/uk/co/nickbutcher/adaptiveiconplayground/MainActivity.kt (hunk 1 above)
Comment (TarasMazepa) [reply app/src/main/java/uk/co/nickbutcher/adaptiveiconplayground/MainActivity.kt]: done

**Comment by TarasMazepa** [comment]
I signed it!

**Comment by TarasMazepa** [comment]
Hey, please merge!

**Comment by TarasMazepa** [comment]
@nickbutcher can you please merge this change?

**Pull request closed without merging**